python/glow/hail/functions.py

```diff
@@ -156,6 +154,8 @@ def from_matrix_table(mt: MatrixTable, include_sample_ids: bool = True) -> DataF
     """
     Converts a Hail MatrixTable to a Glow DataFrame.
```

Comment:
Could you describe what the schema will look like? How is it translated from the Hail schema?